src/Libraries/Microsoft.Extensions.AI.Ollama/README.md

```diff
@@ -1,280 +1,11 @@
-
-var embeddings = await generator.GenerateAsync("What is AI?");
-
-Console.WriteLine(string.Join(", ", embeddings[0].Vector.ToArray()));
-```
-
-### Text embedding generation with caching
-
-```csharp
-using Microsoft.Extensions.AI;
-using Microsoft.Extensions.Caching.Distributed;
-using Microsoft.Extensions.Caching.Memory;
-using Microsoft.Extensions.Options;
-
-IDistributedCache cache = new MemoryDistributedCache(Options.Create(new MemoryDistributedCacheOptions()));
-
-IEmbeddingGenerator<string, Embedding<float>> ollamaGenerator =
-    new OllamaEmbeddingGenerator(new Uri("http://localhost:11434/"), "all-minilm");
-
-IEmbeddingGenerator<string, Embedding<float>> generator = new EmbeddingGeneratorBuilder<string, Embedding<float>>(ollamaGenerator)
-    .UseDistributedCache(cache)
-    .Build();
-
-foreach (var prompt in new[] { "What is AI?", "What is .NET?", "What is AI?" })
-{
-    var embeddings = await generator.GenerateAsync(prompt);
-
-    Console.WriteLine(string.Join(", ", embeddings[0].Vector.ToArray()));
-}
-```
-
-### Dependency Injection
-
-```csharp
-using Microsoft.Extensions.AI;
-using Microsoft.Extensions.DependencyInjection;
-using Microsoft.Extensions.Hosting;
-using Microsoft.Extensions.Logging;
-
-// App Setup
-var builder = Host.CreateApplicationBuilder();
-builder.Services.AddDistributedMemoryCache();
-builder.Services.AddLogging(b => b.AddConsole().SetMinimumLevel(LogLevel.Trace));
-
-builder.Services.AddChatClient(new OllamaChatClient(new Uri("http://localhost:11434/"), "llama3.1"))
-    .UseDistributedCache()
-    .UseLogging();
-
-var app = builder.Build();
-
-// Elsewhere in the app
-var chatClient = app.Services.GetRequiredService<IChatClient>();
-Console.WriteLine(await chatClient.GetResponseAsync("What is AI?"));
-```
-
-### Minimal Web API
-
-```csharp
-using Microsoft.Extensions.AI;
-
-var builder = WebApplication.CreateBuilder(args);
-
-builder.Services.AddChatClient(
-    new OllamaChatClient(new Uri("http://localhost:11434/"), "llama3.1"));
-
-builder.Services.AddEmbeddingGenerator(new OllamaEmbeddingGenerator(endpoint, "all-minilm"));
-
-var app = builder.Build();
-
-app.MapPost("/chat", async (IChatClient client, string message) =>
-{
-    var response = await client.GetResponseAsync(message, cancellationToken: default);
-    return response.Message;
-});
-
-app.MapPost("/embedding", async (IEmbeddingGenerator<string,Embedding<float>> client, string message) =>
-{
-    var response = await client.GenerateAsync(message);
-    return response[0].Vector;
-});
-
-app.Run();
-```
-
-## Feedback & Contributing
-
-We welcome feedback and contributions in [our GitHub repo](https://github.com/dotnet/extensions).
+* [OllamaSharp](https://www.nuget.org/packages/OllamaSharp)
+* [Microsoft.Extensions.AI](https://www.nuget.org/packages/Microsoft.Extensions.AI)
+* [Microsoft.Extensions.AI.Abstractions](https://www.nuget.org/packages/Microsoft.Extensions.AI.Abstractions)
```

Comment:
I'm leaving the related packages on the M.E.AI.Ollama package README, but dropping them from the others.